Sources/NIOOpenSSL/SSLConnection.swift

```diff
@@ -49,22 +48,9 @@ internal final class SSLConnection {
         return false
     }
     
-    init? (_ ssl: UnsafeMutablePointer<SSL>, parentContext: SSLContext) {
-        self.ssl = ssl
+    init (_ ownedSSL: UnsafeMutablePointer<SSL>, parentContext: SSLContext) {
```

Comment:
we should make the `ownedSSL` the name that is visible from the outside, otherwise it doesn't carry its meaning as well

Reply:
also: extra space between `init` and `(`